### DISCUSSION TITLE
Explorer Patcher will not update or uninstall

### DISCUSSION THREAD

**Discussion by mmschmieg** [valinet/ExplorerPatcher#2199] [General]
I have been trying Explorer Patcher since the March Windows 11 Backup messed with the Taskbar and it has been working well. However, today, it tells me there is a new version, 22621.141354.3, but it will not install. I have followed the instructions in the error message to no avail. I attempted to uninstall Explorer Patcher, but it gives me the same error message and fails. I do not know how to recover from this problem. Every time I boot or make a change in the EP settings, it gives me the notification about the update. I am at a complete loss here about how to proceed short of restoring the system from a ten day old backup.

Edition	Windows 11 Pro
Version	22H2
Installed on	‎10/‎1/‎2022
OS build	22621.1413
Experience	Windows Feature Experience Pack 1000.22639.1000.0

Explorer Patcher Version 22621.1413.54.1

**Comment by Faks** [comment]
try clicking alt+tab multiple times and it should crash.

**Comment by valinet** [comment]
Have you tried manually deleting the files?

**Comment by effemmeffe** [comment]
Same issue here. I'm about to completely remove it by deleting the files, because any other attempt to fix this or even uninstall fails.

**Comment by ghost** [comment]
@mmschmieg 
First, you can set not to be notified for updates! I did it.
To uninstall: try 2-3 times to uninstall and restart the computer, if it say "Let the running uninstall-routine finish and try again" (or similar); it could be, that EP is still present in control-panel and it say "EP has been already uninstalled. Do you want to cancel this entry?"
The you can install the last version of EP and set updates not to be notified about.
In the future install a new version over the present.

I did this way on some VMs (VirtualBox) and it works; after that I could uninstall and reinstall without issues.
(or similar)

**Comment by mmschmieg** [comment]
Removing the files in Program Files\ExplorerPatcher allowed the update to install on my system. I was hesitant to do that initially as I didn’t know how the system would function without them, so I moved them to an external drive. Once the directory was emptied, the update worked.
 
 
--- Mike Schmieg
 
The government is that great fiction by which everyone endeavors to live at the expense of everyone else. – Frederic Bastiat

**Comment by LarsErikOsterud** [comment]
I have a similar problem. But can't delete the files. They are "locked" (in use).
Neither uninstall or new install works. How do I "unlock" them?

**Comment by mmschmieg** [comment]
> I have a similar problem. But can't delete the files. They are "locked" (in use). Neither uninstall or new install works. How do I "unlock" them?

If you cannot exit Patcher, try booting to safe mode, then deleting the files. That should work as Patcher won't load in safe mode.

**Comment by MingHejin** [comment]
Dear friend, let me tell you how to uninstall it.
Please rename the installation program, "install", to uninstall. "
Which is renamed as ep_uninstall.exe I hope I can help you.





等待去年来临
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "valinet/ExplorerPatcher"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年7月3日(星期一) 上午10:12
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;Re: [valinet/ExplorerPatcher] Explorer Patcher will not update or uninstall (Issue #1136)





  
I have a similar problem. But can't delete the files. They are "locked" (in use). Neither uninstall or new install works. How do I "unlock" them?
  
If you cannot exit Patcher, try booting to safe mode, then deleting the files. That should work as Patcher won't load in safe mode.
